jest.config.js

```diff
@@ -3,12 +3,13 @@ module.exports = {
   coverageDirectory: 'jest-coverage/',
   coverageThreshold: {
     global: {
-      branches: 6.94,
-      functions: 8.85,
-      lines: 11.76,
-      statements: 11.78,
+      branches: 21.24,
+      functions: 23.01,
+      lines: 27.19,
+      statements: 27.07,
     },
   },
   setupFiles: ['./test/setup.js', './test/env.js'],
+  setupFilesAfterEnv: ['./test/jest/setup.js'],
```

Comment:
How does this differ from setupFiles? Can we not set up jest-dom in one of our existing files?

Reply:
I've tried to load the file in `setupFiles`, but I'm getting this error for many tests:
![image](https://user-images.githubusercontent.com/80175477/115466134-7efc2000-a1e4-11eb-99cc-09343a06bb69.png)

With `setupFilesAfterEnv` it works correctly. `setupFilesAfterEnv` is also mentioned on the official react testing library site: https://github.com/testing-library/jest-dom#usage